moveit_ros/planning/planning_scene_monitor/src/planning_scene_monitor.cpp

```diff
@@ -181,6 +181,19 @@ PlanningSceneMonitor::~PlanningSceneMonitor()
   rm_loader_.reset();
 }
 
+planning_scene::PlanningScenePtr PlanningSceneMonitor::copyPlanningScene(const moveit_msgs::PlanningScene& diff)
+{
+  planning_scene::PlanningScenePtr scene;
+  {
+    SingleUnlock lock(this, true);
+    scene = getPlanningScene()->diff();
```

Comment:
Shouldn't this suffice:
```suggestion
    scene = LockedPlanningSceneRO(this)->diff();
```